gateways/orbital/request_builders_test.go

```diff
@@ -120,6 +124,37 @@ func TestBuildAuthRequest(t *testing.T) {
 				},
 			},
 		},
+		{
+			"Auth with applepay",
+			&applepayBase,
+			Request{
+				Body: RequestBody{
+					OrbitalConnectionUsername: "username",
+					OrbitalConnectionPassword: "password",
+					MerchantID:                1,
+					IndustryType:              IndustryTypeEcomm,
+					MessageType:               MessageTypeAuth,
+					BIN:                       BINStratus,
+					TerminalID:                TerminalIDStratus,
+					XMLName:                   xml.Name{Local: RequestTypeNewOrder},
+					AccountNum:                mastercardBase.CreditCard.Number,
```

Comment:
should we be checking applepayBase here?

Reply:
oops yeah I guess it doesnt really matter since w edont change anything but good for consistency